app/assets/stylesheets/local/footer.scss

```diff
@@ -89,42 +89,45 @@
 
 }
 
-$accept-cookies-banner-container-height: 2.8125rem;
-
+$accept-cookies-banner-tall: 5rem;
+$accept-cookies-banner-reg:  3rem;
 nav.accept-cookies-banner-nav {
+  border-top: 2px solid lightgray;
```

Comment:
Can you use one of our brand grey colors contained in the existing scss variables, instead of an arbitrary grey, so we stick to brand colors? For consistency and policy.

Reply:
Good call!